homework9_files.py

```diff
@@ -0,0 +1,56 @@
+"""
+Homework9.
+
+Description:
+    write script that will read "input.txt" file and find there
+    all characters from english alphabet
+    collect these characters and their positions in file
+    after info collected
+    this info as text write to "output.txt" file in such format
+    ####################
+    <first found letter> -> pos1
+    <next_letter> -> pos2
+    <next_letter -> pos3
+    etc
+    #######################
+"""
+
+import re
+
+fh = None
+letters = []
+input_file = 'input.txt'
+output_file = 'output.txt'
+
+# open input file in read more
+
+try:
```

Comment:
No no no - better to use single try-except  thee use context manger for reading file, other context manager for writing to file. Between context managers your logic with finditer.

If context manager used - no fh.close and no finally block required